File: Framework/CSharp/Framework/Framework/Data/SmartDatabaseType.cs
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
namespace Smartkernel.Framework.Data
{
	/// <summary>
	/// 数据库类型
	/// </summary>
	public enum SmartDatabaseType
	{
		/// <summary>
		/// OleDb数据库
		/// </summary>
		OleDb = 1,

		/// <summary>
		/// SqlServer数据库
		/// </summary>
		SqlServer = 2,

		/// <summary>
		/// SQLite数据库
		/// </summary>
		SQLite = 3,

		/// <summary>
		/// MySql数据库
		/// </summary>
		MySql = 4,

		/// <summary>
		/// Oracle数据库
		/// </summary>
		Oracle = 5,

		/// <summary>
		/// SqlCe数据库
		/// </summary>
		SqlCe = 6,

		/// <summary>
		/// Odbc数据库
		/// </summary>
		Odbc = 7
	}
}

File: Framework/Java/framework/framework/smartkernel/framework/data/SmartDatabaseType.java
/*
 * 项目：Smartkernel.Framework
 * 作者：曹艳白
 * 邮箱：caoyanbai@139.com
 */
package smartkernel.framework.data;

/**
 * 数据库类型
 * 
 */
public enum SmartDatabaseType {
	/**
	 * OleDb数据库
	 */
	OleDb(1),

	/**
	 * SqlServer数据库
	 */
	SqlServer(2),

	/**
	 * SQLite数据库
	 */
	SQLite(3),

	/**
	 * MySql数据库
	 */
	MySql(4),

	/**
	 * Oracle数据库
	 */
	Oracle(5),

	/**
	 * SqlCe数据库
	 */
	SqlCe(6),

	/**
	 * Odbc数据库
	 */
	Odbc(7);

	private int value;

	private SmartDatabaseType(int value) {
		this.value = value;
	}

	/*
	 * (non-Javadoc)
	 * 
	 * @see java.lang.Enum#toString()
	 */
	public String toString() {
		return String.valueOf(this.value);
	}
}
